tools/update_postman.js

```diff
@@ -0,0 +1,58 @@
+
+const axios = require('axios');
+const Converter = require('openapi-to-postmanv2');
+
+const POSTMAN_COLLECTION_ID = 'fdc9fd1f-d7bc-4f7e-b5ce-9189dbcbb81d';
+const POSTMAN_API_KEY = process.env.POSTMAN_API_KEY;
+const openapiUrl = 'https://api.svix.com/api/v1/openapi.json';
```

Comment:
We want to use the prod json not the one stored in the repo so download it.

Reply:
Why? We are doing it whenever the libs are released, let's just do whatever is stored locally.